pkgs/by-name/du/duckstation/package.nix

```diff
@@ -57,26 +63,27 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     SDL2
     curl
+    extra-cmake-modules
     libXrandr
     libbacktrace
     qtbase
     qtsvg
     qtwayland
     wayland
-  ]
-  ++ cubeb.passthru.backendLibs;
+  ] ++ cubeb.passthru.backendLibs;
```

Comment:
Is this diff related to nixpkgs-hammering?

Reply:
Nope, this is nixfmt.